markups/cmark_render.py

```diff
@@ -0,0 +1,38 @@
+# vim: ts=8:sts=8:sw=8:noexpandtab
+
+# This file is part of python-markups module
+# License: BSD
+# Copyright: (C) Shengjing Zhu, 2016
+
+from CommonMark import HtmlRenderer
+from CommonMark.common import escape_xml
+
+
+class CommonMarkRender(HtmlRenderer):
```

Comment:
Why do you want to diverge from upstream CommonMark-py here? Maybe you can submit the highlighting support code to CommonMark-py instead?